base/MAPL_EASEConv.F90

```diff
@@ -0,0 +1,818 @@
+
+module MAPL_EASEConvMod
```

Comment:
We have a convention for module names and file names.  Be sure to follow it.  no `Mod`

I also suggest expanding the abbreviation `Conv`: electrons are cheap.
```suggestion
module MAPL_EASEConversion
```